### DISCUSSION TITLE
文档页的组织顺序

### DISCUSSION THREAD

**Discussion by EndlessPeak** [wangchucheng/hugo-eureka#92] [Q&A] [answered]
你好，我注意到你的文档页面内各文档组织是按照编写时间递增的顺序，即先写的文档排在前后写的文档排在后;
但是我自己写的时候发现我的文档顺序是先写的在后，后写的在前，请问如何修改？

**Comment by wangchucheng** [comment, marked as answer]
文档页面是按照Front Matters中的weight属性的大小进行排序的，而非时间
